tiatoolbox/models/architecture/unet.py

```diff
@@ -183,26 +192,35 @@ def forward(self, x: torch.Tensor):
 class UNetModel(ModelABC):
     """Generate families of UNet model.
 
-    This supports different encoders. However, the decoder is relatively simple-
-    each upsampling block contains a number of vanilla convolution layers, that
-    are not customizable. Additionally, the aggregation between down-sampling and
-    up-sampling is addition, not concatenation.
+    This supports different encoders. However, the decoder is relatively
+    simple- each upsampling block contains a number of vanilla
+    convolution layers, that are not customizable. Additionally, the
+    aggregation between down-sampling and up-sampling is addition, not
+    concatenation.
 
     Args:
-        num_input_channels (int): Number of channels in input images.
-        num_output_channels (int): Number of channels in output images.
-        encoder (str): Name of the encoder, currently supports:
-          - "resnet50": The well-known ResNet50- this is not the pre-activation model.
-          - "unet": The vanilla UNet encoder where each down-sampling level
-            contains 2 blocks of Convolution-BatchNorm-ReLu.
-        decoder_block (list): A list of convolution layers. Each item is an
-          integer and denotes the layer kernel size.
-        classifier (list): A list of convolution layers before the final 1x1
-          convolution. Each item is an integer denotes the layer kernel size. The
-          default is `None` and contains only the 1x1 convolution.
+        num_input_channels (int):
+            Number of channels in input images.
+        num_output_channels (int):
+            Number of channels in output images.
+        encoder (str):
+            Name of the encoder, currently supports:
+            - "resnet50": The well-known ResNet50- this is not the
+              pre-activation model.
+            - "unet": The vanilla UNet encoder where each down-sampling
+              level contains 2 blocks of Convolution-BatchNorm-ReLu.
+        decoder_block (list):
+            A list of convolution layers. Each item is an integer and
+            denotes the layer kernel size.
+        classifier (list):
```

Comment:
check this input.

Reply:
@vqdang  Can you look at this please? It looks like you added this code. I'm not sure what is correct here.

Reply:
Can we remove this? It looks to be unused.

Reply:
@shaneahmed and @dang, can we address this? Can it be removed here?